lambda_src/drivers/process_https.py

```diff
@@ -90,6 +90,15 @@ def process_row(
             req_headers['authorization'] = req_auth['authorization']
         elif 'headers' in req_auth:
             req_headers.update(req_auth['headers'])
+        elif 'body' in req_auth:   
+            if json:
+                raise ValueError(f"auth 'body' key and json param are both present")
```

Comment:
Is it the case that auth body and json should not simultaneously? This seems to be a contract with the GEFF clients and I think should be documented in the wiki along with being present in code.

Reply:
Documented this in the wiki.